tests/test_telcameras_v2.py

```diff
@@ -249,6 +289,45 @@ def test_post_new_record(self):
 
             self.assertEqual(str(pers_aggr.person_id), posted_person_aggregate['personId'])
 
+    def test_post_new_record_with_double_zone(self):
+        """ Test posting a new message with a double zone in the count message """
+        post_data = json.loads(TEST_POST_DOUBLE_ZONE)
+        response = self.client.post(self.URL, post_data, **AUTHORIZATION_HEADER, format='json')
+
+        # Check the Observation record
+        self.assertEqual(response.status_code, 201, response.data)
+        self.assertEqual(Observation.objects.all().count(), 1)
+        observation = Observation.objects.all()[0]
+        fields_to_check = ('sensor', 'sensor_type', 'sensor_state', 'owner', 'supplier', 'purpose', 'latitude',
+                           'longitude', 'interval', 'timestamp_message', 'timestamp_start')
+        for attr in fields_to_check:
+            if type(getattr(observation, attr)) is Decimal:
+                self.assertEqual(float(getattr(observation, attr)), post_data['data'][0][attr])
+            elif type(getattr(observation, attr)) is datetime:
+                self.assertEqual(getattr(observation, attr), parser.parse(post_data['data'][0][attr]))
+            else:
+                self.assertEqual(getattr(observation, attr), post_data['data'][0][attr])
+
+        # Check the CountAggregate records
+        self.assertEqual(CountAggregate.objects.all().count(), len(post_data['data'][0]['aggregate']))
+        for count_aggr in CountAggregate.objects.all():
+
+            # Get the post data for this CountAggregate (they might not be in the same order)
+            posted_count_aggregate = None
+            for postedCountAggregate in post_data['data'][0]['aggregate']:
+                if postedCountAggregate['id'] == count_aggr.external_id:
+                    posted_count_aggregate = postedCountAggregate
```

Comment:
dit is wel nasty. Je gebruikt nu camel casing en snake casing door elkaar heen omdat je onderscheid wilt maken tussen twee vars die ongeveer hetzelfde doen. Een van de twee is een expected value, dat kan je gebruiken om het onderscheid aan te duiden.

Reply:
Idd erg nasty. Ik corrigeer het.